manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
+PackageVersion: 1.5.1
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-04-08

```

```suggestion
ReleaseDate: 2024-04-08
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
The application will not launch if VCRedist is not installed